backend/src/middleware/error.js

```diff
@@ -0,0 +1,4 @@
+module.exports = function(err, req, res, next) {
```

Comment:
If we call this middleware, it will return a 500 error all the time regardless of what happened. In case the input was wrong, we want to return 400 error. We can keep this as a file, but not include it as a middleware on express server for now until we can make it handle all different scenarios.